### ISSUE TITLE
fix: downloads sparkline hover is hidden under mouse

### ISSUE THREAD

**Issue by eseidel** [dart-lang/pub-dev#8301]
When you hover the sparkline the number tracks under your cursor (rather than the line), so you can't move the mouse off from over the number to see it.

<img width="244" alt="image" src="https://github.com/user-attachments/assets/e31f9dc2-f5a8-4a04-89cc-5622f2027829">

Screenshots don't show the mouse cursor, so it's hard to show you the bug.

FYI @szakarias

**Comment by eseidel** [comment]
Ah!  When I capture with the mouse pointer enabled I see the potential bug:
![image](https://github.com/user-attachments/assets/e1d91653-ca31-4e3e-8e9b-f1625bbc6426)


I have mouse cursor zoom enabled (which it appears the screenshot tool does not respect, and the pub.dev code does not account for):

<img width="701" alt="image" src="https://github.com/user-attachments/assets/4160565f-36b6-44a9-a43d-c9cfa9bb68c0">

**Comment by eseidel** [comment]
Here is how it looks taken from my phone camera looking at my screen:
![image](https://github.com/user-attachments/assets/d38f92c2-03a8-4cbc-ab54-407b9f02e724)

**Comment by sigurdm** [comment]
According to https://stackoverflow.com/questions/1889487/get-size-of-mouse-cursor-in-javascript there is no way of querying for the cursor size from a web-context.

We could maybe change the cursor to a size of our liking when hovering the sparkline - but that seems unfriendly to people who chose a larger cursor.

Perhaps we should put the tooltip in a fixed position?

**Comment by eseidel** [comment]
Yeah,I'd just put it in a fixed position.  NPM for example puts it to the side:
<img width="404" alt="image" src="https://github.com/user-attachments/assets/4c1bc746-09da-41c3-9358-831b7f137dee">

**Comment by eseidel** [comment]
Github puts it above the cursor: 
<img width="399" alt="image" src="https://github.com/user-attachments/assets/de229695-f322-44eb-b923-2aa39365fbff">

**Comment by eseidel** [comment]
Anyway, I'm not a UI designer, I defer to folks with much more design ability than myself.  Just wanted to report that at least in my setup I couldn't read the numbers. Thanks!

**Comment by dishankjindal1** [comment]
I think having it above the cursor makes much more sense.

**Comment by sigurdm** [comment]
@szakarias what do you think - should we move the sparkline hover above?
